shell/probe/raindrop.ps1

```diff
@@ -33,13 +33,24 @@ while ($true) {
             "dat" = $dat
             "version" = "2.1"
```

Comment:
This needs to get bumped.

Reply:
also https://github.com/preludeorg/service/blob/main/functions/probe_tasks/app.py#L11